src/core/link-to-dfn.js

```diff
@@ -24,12 +26,13 @@ export function run(conf, doc, cb) {
         if (oldIsDfn && newIsDfn) {
           // Only complain if the user provides 2 <dfn>s
           // for the same term.
+          dfn.attr('id',`${title + count}`);
```

Comment:
You might end up with strange IDs here... like `"foo bar1"`. You can use the `dfn.makeID(null, title)` here to solve that.